index.html

```diff
@@ -6,8 +6,67 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7cbe454c49.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <header class="header-container">
+    <div class="sub-container">
+        <a href="#" class="logo-cont">Olamarx</a>
+      <button type="submit" class="header-font"><i class="fa fas-solid fa-bars font"></i> </button> 
+    </div>
+    </header>
+
+    <section>
+        <div class="main-container">
+            <div class="images">
+
+                <div class="img left-img">
+                    <img src="./images/Rectangle6.png" alt="Rectangle">
+                    <img src="./images/Ellipse11.png" alt="Ellipse">
+                </div>
+
+                <div class="img center-img">
+                    <img src="./images/Rectangle7.png" alt="Rectangle">
+                    <img src="./images/Ellipse5.png" alt="Ellipse">
+                </div>
+
+                <div class="img rigth-img">
+                    <img src="./images/Rectangle4.png" alt="Rectangle">
+                    <img src="./images/Ellipse3.png" alt="Ellipse">
+                </div>
+
+            </div>
+        </div>
```

Comment:
- [ ] Please make sure the background images uses a proper background images included through the stylesheet by using the `background-image` property